### PR TITLE
OhemLoss:  bug fix

### PR DESCRIPTION
**Pull request by danielafrimi**
OhemLoss forward, if all pixels are ignore labels, return empty loss tensor and pass it to logits device

### CODE REVIEW

**Comment by BloodAxe** [comment]
@danielafrimi Thanks! But please fix the PR using [signed commits](https://github.com/Deci-AI/super-gradients/blob/master/CONTRIBUTING.md#signed-commits) otherwise we would not be able to merge your pr
